### ISSUE TITLE
Percentage Filter

### ISSUE THREAD

**Issue by dderjoel** [freearhey/vue2-filters#84]
I was wondering if a small percentage Filter would be an useful addition: 
currently, we use in our project.
```
export const percentFilter = (value = 0.0, decimals = 0): string => {
  return `${(value * 100).toFixed(decimals)} %`;
};
```
I know its just a very small addition, but it will make (at least my) code base cleaner, since I can use `formatter: (v: number): string => this.$options.filters.percent(v)` 

feel free to make suggestions

**Comment by freearhey** [comment]
@dderjoel I'm sorry for such a long delay, but I finally found some free time and added this filter to the package.

I've already published the beta version `v0.10.0-beta.1` and it would be great if you could test it. And if there are no bugs, then I'll make the final release.

**Comment by dderjoel** [comment]
Works well, thank you for adding it!
